src/localStorageService.ts

```diff
@@ -29,6 +42,35 @@ const localStorageService = {
     const updatedHidden = { ...parseHidden }
     updatedHidden[key] = value
     localStorageService.setItem(SAVED_HIDDEN, JSON.stringify(updatedHidden))
+  },
+  updateResized(category: string, resizing: ColumnResizing) {
```

Comment:
don't like the approach when we keep the logic of one feature in different places. let's discuss this when we have time, maybe we find something better